winlogtimeline/util/project.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, project_file_path):
 
             # Set up the directory structure
             self._path = project_directory
-            self._log_file = 'logs.sqlite'
+            self._file_name = self._path.split('/')[-1]
```

Comment:
Directly modifying the path is a good way to introduce OS compatibility issues. In this case, '/' is only a valid path separator on Unix-based systems, not on Windows. 

There are quite a few ways to fix this, all using the `os.path` module. I'd recommend using `os.path.dirname`.

Reply:
Actually, now that I think about it, it'd be better to have the database file match the .elv filename instead of the directory name. `os.path.basename` would be better for that.

Reply:
I figured that's what you meant.